tests/test_logger.py

```diff
@@ -14,16 +15,24 @@ class LoggingHandlerTest(unittest.TestCase):
 
     def setUp(self):
         self.qapp = QApplication([])
+        self.logdir = "logfile-unittest"
```

Comment:
How about `log_dir`?

Reply:
Please add a trailing "/" to show that the path is a directory explicitly.

Reply:
Also, I think the directory path should be a full path!

Reply:
I thought that Unit test is executed at the location of `iquip`.
So I make another directory for dummy log file at the `iquip`, like `iquip/logfile-unittest` and delete that directory.
I don't know the exact directory where unittest is executed. 

Setting full path as `os.getcwd()+self.log_dir` would be same as previous implementation. Do you prefer this way? Or did I missed something?

Reply:
As @Aijuh and I discussed yesterday, the current implementation doesn't seem to matter, so it is okay to remain it as it is.

Please just add a trailing "/"!